File: php-sdk/src/v1/client/operate/OperateRequest.php
<?php

namespace triplink\vcc\card\client\operate;


class OperateRequest {

    var $requestId;

    var $requestTime;

    var $requestType;

    var $subRequestType;

    var $merchantName;

    var $channelType;

    var $cardInfo;

    var $riskInfo;

    var $requestSource;

    var $operator;

    var $reqExtra;
    
    var $sign;

    public function getRequestId()
    {
        return $this->requestId;
    }

    /**
     * @return the $requestTime
     */
    public function getRequestTime()
    {
        return $this->requestTime;
    }

    /**
     * @return the $requestType
     */
    public function getRequestType()
    {
        return $this->requestType;
    }

    /**
     * @return the $subRequestType
     */
    public function getSubRequestType()
    {
        return $this->subRequestType;
    }

    /**
     * @return the $merchantName
     */
    public function getMerchantName()
    {
        return $this->merchantName;
    }

    /**
     * @return the $channelType
     */
    public function getChannelType()
    {
        return $this->channelType;
    }

    /**
     * @return the $cardInfo
     */
    public function getCardInfo()
    {
        return $this->cardInfo;
    }

    /**
     * @return the $riskInfo
     */
    public function getRiskInfo()
    {
        return $this->riskInfo;
    }

    /**
     * @return the $requestSource
     */
    public function getRequestSource()
    {
        return $this->requestSource;
    }

    /**
     * @return the $operator
     */
    public function getOperator()
    {
        return $this->operator;
    }


    /**
     * @return the $reqExtra
     */
    public function getReqExtra()
    {
        return $this->reqExtra;
    }

    /**
     * @param field_type $requestId
     */
    public function setRequestId($requestId)
    {
        $this->requestId = $requestId;
    }

    /**
     * @param field_type $requestTime
     */
    public function setRequestTime($requestTime)
    {
        $this->requestTime = $requestTime;
    }

    /**
     * @param field_type $requestType
     */
    public function setRequestType($requestType)
    {
        $this->requestType = $requestType;
    }

    /**
     * @param field_type $subRequestType
     */
    public function setSubRequestType($subRequestType)
    {
        $this->subRequestType = $subRequestType;
    }

    /**
     * @param field_type $merchantName
     */
    public function setMerchantName($merchantName)
    {
        $this->merchantName = $merchantName;
    }

    /**
     * @param field_type $channelType
     */
    public function setChannelType($channelType)
    {
        $this->channelType = $channelType;
    }

    /**
     * @param field_type $cardInfo
     */
    public function setCardInfo($cardInfo)
    {
        $this->cardInfo = $cardInfo;
    }

    /**
     * @param field_type $riskInfo
     */
    public function setRiskInfo($riskInfo)
    {
        $this->riskInfo = $riskInfo;
    }

    /**
     * @param field_type $requestSource
     */
    public function setRequestSource($requestSource)
    {
        $this->requestSource = $requestSource;
    }

    /**
     * @param field_type $operator
     */
    public function setOperator($operator)
    {
        $this->operator = $operator;
    }


    /**
     * @param field_type $reqExtra
     */
    public function setReqExtra($reqExtra)
    {
        $this->reqExtra = $reqExtra;
    }
    /**
     * @return the $sign
     */
    public function getSign()
    {
        return $this->sign;
    }

    /**
     * @param field_type $sign
     */
    public function setSign($sign)
    {
        $this->sign = $sign;
    }

    
    

    
}
?>

File: src/main/java/com/ctrip/ccard/creditcard/vcc/bean/V1/OperateRequest.java
package com.ctrip.ccard.creditcard.vcc.bean.V1;

/**
 * operate接口 入参实体类
 */
public class OperateRequest {

    /**
     * 请求流水号
     */
    private String requestId;
    /**
     * 请求时间
     * 格式 ：yyyyMMddHHmmss
     */
    private String requestTime;
    /**
     * 请求类型
     * CREATE 创建卡，UPDATE 更新卡，CLOSE 关闭卡
     */
    private String requestType;
    /**
     * 请求子类型
     * CARD_EXTERNAL 虚拟卡
     */
    private String subRequestType;
    /**
     * 请求商户名 服务方预先分配
     */
    private String merchantName;
    /**
     * 发卡通道类型 服务方预先分配
     */
    private String channelType;
    /**
     * 卡信息
     */
    private CardInfo cardInfo;
    /**
     * 风控信息
     */
    private RiskVerifyInfo riskInfo;
    /**
     * 请求来源标注  请求方自定义区分
     */
    private String requestSource;
    /**
     * 调用端操作人
     */
    private String operator;
    /**
     * 签名信息
     */
    private String sign;
    /**
     * 请求扩展字段
     * 若涉及扩展域使用，请参考“请求扩展域-字段说明”
     * json格式  示例 {"BinFlagDcre":"12345"}
     */
    private String reqExtra;

    public String getRequestId() {
        return requestId;
    }

    public void setRequestId(String requestId) {
        this.requestId = requestId;
    }

    public String getRequestTime() {
        return requestTime;
    }

    public void setRequestTime(String requestTime) {
        this.requestTime = requestTime;
    }

    public String getRequestType() {
        return requestType;
    }

    public void setRequestType(String requestType) {
        this.requestType = requestType;
    }

    public String getSubRequestType() {
        return subRequestType;
    }

    public void setSubRequestType(String subRequestType) {
        this.subRequestType = subRequestType;
    }

    public String getMerchantName() {
        return merchantName;
    }

    public void setMerchantName(String merchantName) {
        this.merchantName = merchantName;
    }

    public String getChannelType() {
        return channelType;
    }

    public void setChannelType(String channelType) {
        this.channelType = channelType;
    }

    public CardInfo getCardInfo() {
        return cardInfo;
    }

    public void setCardInfo(CardInfo cardInfo) {
        this.cardInfo = cardInfo;
    }

    public RiskVerifyInfo getRiskInfo() {
        return riskInfo;
    }

    public void setRiskInfo(RiskVerifyInfo riskInfo) {
        this.riskInfo = riskInfo;
    }

    public String getRequestSource() {
        return requestSource;
    }

    public void setRequestSource(String requestSource) {
        this.requestSource = requestSource;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public String getSign() {
        return sign;
    }

    public void setSign(String sign) {
        this.sign = sign;
    }

    public String getReqExtra() {
        return reqExtra;
    }

    public void setReqExtra(String reqExtra) {
        this.reqExtra = reqExtra;
    }
}
